oscarlab/graphene
# [LibOS] test/apps: Remove unused files

## Description of the changes 

This is a preparation to remove LibOS/shim/test/apps submodule (either by merging it back or by detaching it from this repo). It removes almost 25 MB in unused files.

## How to test this PR? 

If I didn't miss any reference, nothing should change.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1382)
